sklearn/tests/test_metaestimators.py

```diff
@@ -15,6 +15,7 @@
 from sklearn.feature_selection import RFE, RFECV
 from sklearn.ensemble import BaggingClassifier
 from sklearn.exceptions import NotFittedError
+from sklearn.semi_supervised.self_training import SelfTrainingClassifier
 
 
```

Comment:
Changes in this file are from #12088, right?

I don't understand why they show up in the diff...

Maybe merge master into your branch?

Reply:
Yes, some changes are from that PR. I cherry-picked the commit from the PR from master since the issue also affected my tests.

Reply:
Ok, now that the PR has been merged you should definitely merge master here.